SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -350,6 +354,7 @@ void FSpatialGDKEditorToolbarModule::AddMenuExtension(FMenuBuilder& Builder)
 #endif
 		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().CreateSpatialGDKSchema);
 		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().CreateSpatialGDKSnapshot);
+		Builder.AddMenuEntry(FSpatialGDKEditorToolbarCommands::Get().GenerateTestMaps);
```

Comment:
Nope, this goes into the `Window` menu dropdown:
![image](https://user-images.githubusercontent.com/32096431/108344056-944cd200-71d4-11eb-83d5-96d61ca6352a.png)

Reply:
Nice.
Also, I guess it will come up in the small meeting soon, but we could make it so the testgyms project tries to gen the maps on module load, if that's possible?

Reply:
Oh yeah, that would be really nice.

Reply:
Decided that for now having a button is better since there were additional complications with generating on module load.

Reply:
was wondering if we should isolate this to the testgyms project? somehow...

Reply:
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmm, the commandlet is in the core GDK, though
We could hide it behind a spatialgdksettings bool, and set that bool to true in the testgyms.
but I agree this is kind of really annoying, for not a lot of benefit.
If we can convince ourselves that users won't question this / shoot themselves in the foot, then we are fine.